packages/button/stories/index.ts

```diff
@@ -98,15 +98,14 @@ export const makeOverBackground =
         const color =
             variant === 'black'
                 ? 'rgb(181, 209, 211)'
-                : 'var(--spectrum-global-color-seafoam-600)';
+                : 'var(--spectrum-seafoam-900)';
         return html`
             <div
                 style="
                     --mod-actionbutton-static-content-color: ${color};
                     --mod-button-static-content-color: ${color};
                     background-color: ${color};
-                    color: ${color};
```

Comment:
is color: ${color}; removal deliberate?

Reply:
Yes. @hunterloftis caught an issue where this caused important story text content to appear "hidden" in that is displayed as the same color as the background.